src/style.css

```diff
@@ -1,3 +1,81 @@
+:root {
+  --font-family: 'Roboto', Arial, sans-serif;
+}
+
 body {
+  font-family: var(--font-family);
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  margin: 0;
+
+  --header-height: 60px;
+  --blue-color: #00acdc;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```

Comment:
Don't fix container size (if there is no such a requirement). Here, you've set padding on the .header class which might not be necessary or could be handled differently if the content should dictate the size.